apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -17,6 +17,9 @@ import {
 import { FetchStatus } from "../../helpers/use-fetch";
 import { Alert } from "../../__v3api";
 
+const isStopPageAlert = ({ effect }: Alert): boolean =>
```

Comment:
nitpick: could probably add this to `alerts.ts`, next to the maddeningly similar `isDiversion` function or the similarly single-use `isAmenityAlert` function...

Reply:
A lot of functions in the `alert.ts` file should be renamed to be clearer/more consistent.  We have functions like `alertsByRoute` and `alertsForStop` that do similar things, but are worded weird.